infrateam_first_responder.md

```diff
@@ -78,7 +78,7 @@ Status check from the server (ssh into the prod server for that project and then
 There are applications that need to be deployed separately (i.e., not using `sdr-deploy`):
 
 * **hydra_etd `uat` environment**: deploy via `cap uat deploy` in `hydra-etd`
-* **sul-pub environments beyond `qa`, `stage`, and `prod`**: deploy via `cap ENV deploy` ([ENV values](https://github.com/sul-dlss/sul_pub/tree/master/config/deploy))
+* **sul-pub environments beyond `qa`, `stage`, and `prod`**: deploy via `cap ENV deploy` ([ENV values](https://github.com/sul-dlss/sul_pub/tree/main/config/deploy))
```

Comment:
if we no longer have anything but qa, stage and prod, we should keep this.  if there is still `uat` or something, approving.

Reply:
we do still have a weirdo. `dev`.